deploy/prod/remote-scripts/purge.sh

```
@@ -1,3 +1,4 @@
 echo "Uninstalling docker versions, old and new..."
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)